### ISSUE TITLE
cpu inferance not work well?

### ISSUE THREAD

**Issue by enozhu** [mozilla/LPCNet#58]
when i use nnet_data.c download directly
 from https://media.xiph.org/lpcnet/data/lpcnet_data-7d8d216.tar.gz，cpu inferance work well，outpcm sounds good,
but downloads model from https://jmvalin.ca/misc_stuff/lpcnet_models/， 
and use "python3 src/dump_lpcnet.py {model_name}.h5" to dump to src/nnet_data.c, outpcm sounds noise, but the h5 model sounds good on GPU inferance， why？use the h5 model trained myself has the same problem，possible dump_lpcnet.py work not well in my environment？thanks for answer

**Comment by enozhu** [comment]
solved the problem, because of difference of common gru and cudda gru

**Comment by dalvlv** [comment]
@enozhu Hi，I met the same problem and can  you share more details about your solutions? I notice that 'train_lpcnet.py' use 'cudnngru' but 'test_lpcnet.py'  use 'gru' as default. I tried both and they sounds nearly the same. For the c file of test_lpcnet how I could change and  I doubt why the cpu intferance made bad audio.

**Comment by carlfm01** [comment]
@dalvlv  Did you exported the model and recompile using you generated `nnet_data.c` and `nnet_data.h`?

**Comment by dalvlv** [comment]
Yes, I  mv the new nnet_data.c and nnet_data.h to 'src', and then  recompile the project. But the c inferance sounds bad. I don't know  why.

**Comment by dalvlv** [comment]
@carlfm01 Solved it. I made a mistake in recomplie. Thank you for your  kind remind.

**Comment by ysujiang** [comment]
@ dalvlv

> @carlfm01 Solved it. I made a mistake in recomplie. Thank you for your kind remind.

Does this mean that the results of GPU and CPU are inconsistent? How did you solve it? Can you tell me ?

**Comment by wangshiyao-1119** [comment]
> Yes, I mv the new nnet_data.c and nnet_data.h to 'src', and then recompile the project. But the c inferance sounds bad. I don't know why.

Hello, I would like to ask, why when I generate nnet.c and nnet.h directly in the src folder, then I will make an error when I make lpcnet_demo, what's the matter?

**Comment by chengshaodi** [comment]
@wangshiyao-1119 请问，你解决这个问题了吗？
